common.sh

```diff
@@ -36,6 +36,10 @@ INT_IF=${INT_IF:-}
 #Root disk to deploy coreOS - use /dev/sda on BM
 ROOT_DISK_NAME=${ROOT_DISK_NAME-"/dev/sda"}
 
+for IMAGE_VAR in IRONIC_IMAGE IRONIC_INSPECTOR_IMAGE IPA_DOWNLOADER_IMAGE COREOS_DOWNLOADER_IMAGE ; do
+    KNI_INSTALL_FROM_GIT=1
```

Comment:
nit: for consistency, we use `true` everywhere else, but really any value works

Reply:
will do, I also forgot a line here to conditionally set KNI_INSTALL_FROM_GIT based on the value of IMAGE_VAR, will fix

Reply:
Oh, heh that's certainly the more important thing :smiley: